Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -92,15 +89,10 @@ public async Task ReadUserMessages(DiscordRequest request, Contexts contexts)
         }
 
         [AdminCommand]
-        public async Task SetRoleAsSafe(SetRoleCommand setRoleCommand, Contexts contexts)
+        public async Task SetRoleAsSafe(SetRoleCommand command, Contexts contexts)
         {
-            var roles = setRoleCommand.Roles;
-            if (roles.Count == 0 || !(setRoleCommand.Safe || setRoleCommand.Unsafe))
-            {
-                throw new NotEnoughArgumentsException();
-            }
-            var shouldSetToSafe = setRoleCommand.Safe;
-            await this._rolesService.SetRolesAsSafe(contexts, roles, shouldSetToSafe);
+            var shouldSetToSafe = command.Safe;
```

Comment:
"should" raczej powinno się stosować do testów, lepsze byłoby coś w style isSuccessfulySetToSafe czy coś tego rodzaju

Reply:
ok poprawie.